Mesh_3/examples/Mesh_3/mesh_hybrid_mesh_domain.cpp

```diff
@@ -25,7 +25,7 @@ typedef CGAL::Exact_predicates_inexact_constructions_kernel K;
 typedef K::FT FT;
 typedef K::Point_3 Point;
 typedef FT (Function)(const Point&);
```

Comment:
Don't need this typedef anymore as far as I see.

And missing a test for the deprecated implicit domain.

Reply:
The deprecated `Implicit_mesh_domain_3` is still tested in `test/Mesh_3/test_backward_compatibility.cpp`.

Done for the typedef.

Reply:
We usually don't keep them in the classified reference manual. If you want to keep, what about adding _(deprecated)_ right after?

Reply:
Sure that's a good idea (except maybe put it before)

Reply:
I have updated [my feature submission "V2"][V2], with the commit  a67ac9d8207.

[V2]: https://cgal.geometryfactory.com/~lrineau/Mesh_3_new_API_of_Labeled_mesh_domain_3-V2/Mesh_3/group__PkgMesh__3.html